read.go

```diff
@@ -46,7 +46,9 @@ func (i *UI) read(opts *readOptions) (string, error) {
 				resultErr = fmt.Errorf("failed to read the input: %s", err)
 			}
 
-			resultStr = strings.TrimSuffix(line, "\r\n")
+			resultStr = strings.TrimSuffix(line, LINE_SEP)
```

Comment:
The problem with the initial PR implementation was it then _only_ worked for windows, so was getting the following error when testing on OSX 
![image](https://cloud.githubusercontent.com/assets/3196313/24583927/02c3318a-1729-11e7-949c-0af98f7a3a31.png)